### PR TITLE
Milis Linux support

### PR DESCRIPTION
**Pull request by milisarge**

### CODE REVIEW

**Comment by ben-albrecht** [comment]
@milisarge - Thanks for your contribution. Due to the upcoming Chapel 1.21 release, we probably won't have a chance to look at this until after the release. Thanks for your patience.

**Comment by milisarge** [comment]
@ben-albrecht  :up:

**Comment by ben-albrecht** [comment]
@milisarge - I think @jhoole (or someone from their team) will be taking a look at this after releasing Chapel 1.22 (https://chapel-lang.org/events.html).

File: README.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ each OS.
 
 ## Packages
 
+The following distributions has deployment around the Chapel 1.21.0(git) release.
```
Comment (jhoole) [inline README.md]: Since we're past the 1.21 and 1.22 releases, can you update this for the 1.23 release?

**Review by jhoole — CHANGES_REQUESTED** [review]
Thank you so much for your patience!

I think with the update in the versioning we should be good. I don't know that the Chapel team will have the capacity to take this over, but if you're willing to own future updates/changes needed, this is a beneficial addition.

File: README.md (hunk 1 above)
Comment (milisarge) [reply README.md]: ok i will update it to 1.23 then inform you

Comment (milisarge) [reply README.md]: @jhoole it is done. Package is ready as https://mls.akdeniz.edu.tr/paketler/chapel%231.23.0-1-x86_64.mps.lz

**Review by jhoole — APPROVED** [review]

**Pull request merged**